packages/rest/src/rest.server.ts

```diff
@@ -614,7 +621,15 @@ export class RestServer extends Context implements Server, HttpServerLike {
    * @param options Options for serve-static
    */
   static(path: PathParams, rootDir: string, options?: ServeStaticOptions) {
-    this.httpHandler.registerStaticAssets(path, rootDir, options);
+    if (this._httpHandler) {
+      this.httpHandler.registerStaticAssets(path, rootDir, options);
```

Comment:
I find this a bit weird - you are checking for the existence of `this._httpHandler` and then calling `this.httpHandler.registerStaticAssets`. Shouldn't you use the same property in both places?